### ISSUE TITLE
[주제 2] 데이터 길이

### ISSUE THREAD

**Issue by Phaskal** [DatathonInfo/H.D.A.I.2021#30]
주제 2의 데이터의 경우 디코딩을 실시할 경우 몇몇 파일이 다른 파일과 time stamp 길이가 다르거나 아예 lead자체가 없는 경우가 있습니다. 파일 크기만 확인해봐도 몇 데이터가 다른 데이터에 비해 크기가 압도적으로 적음을 알 수 있습니다. 주로 arrythmia 데이터에서 그런 경우가 있는데 이런 파일들을 그저 이상한 ECG파일이라고 단정 짓고, 데이터 길이가 맞지 않으면 인공지능 모델로 평가하지 않고 arrythmia라고 predict해도 되는지 여쭙고 싶습니다.

**Comment by DatathonInfo** [comment]
안녕하세요 참가자님,

해당 문의에 답변드립니다.

제공해 드린 원천 데이터는 인공지능 학습용 데이터 구축사업을 통해 제작된 데이터입니다.
본 대회의 목적은 구축된 원천 데이터를 통한 활용에 있어, 향후 개방될 원천 데이터를 그대로 제공해 드리고 있습니다.

제출 후 모델 평가가 진행되는 Test Set은 모두 정상적으로 lead가 포함되어 있는 데이터들로 구성되어 있습니다.
현재 문제가 확인된 파일;

Training Set;
![image](https://user-images.githubusercontent.com/92664643/144557734-e201e63b-7f0a-4d32-84c5-0ac138259271.png)

Validation Set;
![image](https://user-images.githubusercontent.com/92664643/144557685-ad8153bc-fcd3-4b29-a5cd-1e28a9a1d755.png)

별도로, 주관 측에 요청하여 문제되는 파일들을 확인중에 있습니다.

위에 제공된 문제 파일 외 문제가 되는 파일이 확인되시면 파일 제거 후 학습을 진행해 주시면 감사하겠습니다.

감사합니다.
H.D.A.I. 2021 운영팀 드림
